src/main/resources/static/assets/js/sdh.js

```diff
@@ -2,18 +2,16 @@
 	new Vue({
 		el: '#contents',
 		data: {
-			fields: [ {key:'show_details', label:' ', headerTitle: 'Show Details', sortable:false}, {key:'domain', sortable:true}, {key:'date_last_reviewed', sortable:true}, {key:'clinician_priority', sortable:true}, {key:'patient_readiness', sortable:true}],
+			fields: [ {key:'show_details', label:' ', headerTitle: 'Show Details', sortable:false}, {key:'domain', sortable:true}, {key:'date_last_reviewed', sortable:true}, {key:'clinician_priority', sortable:true}, {key:'patient_readiness', sortable:true}, {key:'edit', label:' ', headerTitle: 'Edit', sortable:false}],
 			filters: {
 			      domain: '',
 			      date_last_reviewed: '',
 			      clinician_priority: '',
 			      patient_readiness: ''
 			    },
 	        domainOptions: [],
-	        priorityOptions: [],
-	        readinessOptions: [],
-	        reviews: []
-			    
+	        reviews: [],
+	        modalEditRow: { title: '', id: '', domain:'', prioritySelected: '', readinessSelected: '', priorityOptions: [], readinessOptions: [] }
```

Comment:
In an empty row id is a string, but once it's mounted it looks like it is a number. Seems like we should keep the types consistent and make the value null.